packages/color-scale-block/src/components/SquareWithColor.tsx

```diff
@@ -0,0 +1,161 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useDrag } from 'react-dnd';
+import { DragSourceMonitor } from 'react-dnd';
+import { FC, MouseEvent, MouseEventHandler, useState } from 'react';
+import { Tooltip } from '@frontify/fondue';
+import { ItemDragState, TooltipAlignment, TooltipPosition } from '@frontify/fondue';
+import { ColorProps, SquareWithColorProps } from '../types';
+import { DragHandle } from './DragHandle';
+import { CustomizationOptionsModal } from './CustomizationOptionsModal';
+
+const rgbaToHex = (rgba: string, forceRemoveAlpha = false) => {
+    return `#${rgba
+        .replace(/^rgba?\(|\s+|\)$/g, '') // Get's rgba / rgb string values
+        .split(',') // splits them at ","
+        .filter((string: string, index: number) => !forceRemoveAlpha || index !== 3)
+        .map((string: string) => parseFloat(string)) // Converts them to numbers
+        .map((number: number) => number.toString(16)) // Converts numbers to hex
+        .map((string: string) => (string.length === 1 ? `0${string}` : string)) // Adds 0 when length of one number is 1
+        .join('')}`; // Puts the array to together to a string
+};
+
+const getRgbaString = (currentColor: ColorProps) => {
+    if (!(currentColor && currentColor.color)) {
+        return '';
+    }
+
+    return rgbaToHex(
+        `rgba(${currentColor.color.red}, ${currentColor.color.green}, ${currentColor.color.blue}, ${currentColor.color.alpha})`
+    ).toUpperCase();
+};
+
+export const SquareWithColor: FC<SquareWithColorProps> = ({
```

Comment:
We stopped using `FC`. Remove it throughout the app and assign type to the prop object directly.

https://frontify.slack.com/archives/C01JE9QENCF/p1649315672455129

Reply:
Removed all occurrences of `FC`.